mailpoet/lib/Migrator/Runner.php

```diff
@@ -26,18 +26,13 @@ public function __construct(
   }
 
   public function runMigration(string $name): void {
```

Comment:
I wonder if the method should take `$level` argument now as well, or if it's better to rely on the naming.